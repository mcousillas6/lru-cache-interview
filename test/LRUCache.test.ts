import { assert, describe, expect, it } from 'vitest';

describe('LRU Cache', () => {
  it('foo', () => {
    assert.equal(Math.sqrt(4), 2);
  });
});
