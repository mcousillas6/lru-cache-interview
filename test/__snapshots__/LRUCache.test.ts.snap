// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LRU Cache > snapshot 1`] = `
{
  "foo": "bar",
}
`;

exports[`LRU Cache name > snapshot 1`] = `
{
  "foo": "bar",
}
`;

exports[`suite name > snapshot 1`] = `
{
  "foo": "bar",
}
`;
